### DISCUSSION TITLE
How to prevent drawer-content scrolling when drawer is open?

### DISCUSSION THREAD

**Discussion by bayasdev** [saadeghi/daisyui#2134] [Q&A] [answered]
daisyui: `3.2.1`
framework: nextjs

**Comment by saadeghi** [comment]
It doesn't scroll and it shouldn't.  
Can you please share a sample project and also share the name + version of the browser you're using?

**Comment by bayasdev** [comment]
https://github.com/saadeghi/daisyui/assets/49411169/90225b7a-5e8b-4204-8df6-0c04ed9d604c


https://stackblitz.com/edit/daisyui-nextjs-jybazv?file=pages%2Findex.js

Brave Browser: `Version 1.52.130 Chromium: 114.0.5735.198 (Official Build) (arm64)`

**Comment by saadeghi** [comment, marked as answer]
I see. This happens when the sidebar content is not tall enough to scroll.

I created this issue: https://github.com/saadeghi/daisyui/issues/2142  
I will try to find a solution
